### PR TITLE
TC_03.06.013 | Create Multiconfiguration project

### PR DESCRIPTION
**Pull request by LenaDik**
https://trello.com/c/lhKtKD0U/386-tc0306013-new-item-create-multiconfiguration-project

### CODE REVIEW

File: cypress/e2e/createMultiConfigPrj.cy.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+describe('createMultiConfigProject', () => {
+    it('TC_03.06.013 | Create Multiconfiguration project', function() {
+        cy.get('a[href="newJob"]').click();
+        cy.get('input#name').type('ProjectOne');
+        cy.get('li[tabindex="0"] span').contains('Multi-configuration project').click();
+        cy.get('#ok-button').click();
+        cy.get('.jenkins-breadcrumbs__list-item:nth-child(1) > .model-link').click();
+
+        cy.get('td a[href*="job"].jenkins-table__link').should('have.text', 'ProjectOne');
```
Comment (Rom-T) [inline cypress/e2e/createMultiConfigPrj.cy.js:11]: please move `ProjectOne` to fixtures

File: cypress/e2e/createMultiConfigPrj.cy.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/// <reference types="cypress"/>
+
+describe('createMultiConfigProject', () => {
+    it('TC_03.06.013 | Create Multiconfiguration project', function() {
+        cy.get('a[href="newJob"]').click();
+        cy.get('input#name').type('ProjectOne');
```
Comment (Rom-T) [inline cypress/e2e/createMultiConfigPrj.cy.js:6]: please move `ProjectOne` to fixtures

File: cypress/e2e/createMultiConfigPrj.cy.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        cy.get('li[tabindex="0"] span').contains('Multi-configuration project').click();
```
Comment (S-Lanchik) [inline cypress/e2e/createMultiConfigPrj.cy.js:7]: Put 'Multi-configuration project' to fixtures.

**Comment by Paul-Korz** [comment]
Hi. Please close your PR and delete branch.

**Pull request closed without merging**